_posts/2020-01-16-cycle-8-results.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Bisq DAO Cycle 8: Results"
+author: Steve Jain
+excerpt: "Cycle 8 of the Bisq DAO ended at block 609 186 on December 21 2019. This post covers its results. <br><br>"
+en-only: true
+---
+
+This post summarizes the results of Cycle 8 of the Bisq DAO.
+
+It comes a bit later than usual, as Cycle 8 wasn't a pretty one. During the cycle, some significant issues were made apparent in the human side of the network: a lack of management and budgeting resulted in suboptimal resource allotment needed for growth, which resulted in a ballooning BSQ supply, which resulted in an erosion of confidence in BSQ, which ultimately resulted in a sharp downturn in the BSQ price.
+
+In response, key contributors are working on substantial changes to the way Bisq operates in order to address these issues and restore confidence in the Bisq DAO. 
+
+These changes will be discussed more thoroughly in a call next week, as well as in a post on this blog shortly thereafter.
```

Comment:
Great. I'd relax "restore confidence" to something like "ensure continued confidence in the Bisq DAO amongst contributors and users alike". Again, we don't *know* that confidence has been lost, we just know that it soon will be if we don't correct course.